pkg/discovery/network/flannel_test.go

```diff
@@ -0,0 +1,99 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package network_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/submariner-io/submariner-operator/pkg/discovery/network"
+	v1apps "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	v1meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const (
+	NetworkPluginFlannel = "flannel"
```

Comment:
We are defining the list of CNI plugins in submariner repo and using it in operator. I'm fine to merge this PR and update it after the flannel constant is added in the list of CNIs - https://github.com/submariner-io/submariner/blob/85b5c9a7fe57632e75179c7ff55d59ece9cb2f98/pkg/cni/plugins.go#L22

Reply:
I pushed a PR in Submariner repo with the constant - https://github.com/submariner-io/submariner/pull/1941

Reply:
Thanks @sridhargaddam! resolved the other comment.